pages/about.html

```diff
@@ -1,12 +1,42 @@
+<!-- some information about you; interests, background or similar -->
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="/style/about.css" rel="stylesheet"> 
 </head>
 <body>
-  
+  <header>
+    <h1>about</h1>
+      <nav>
+        <ul>
+          <li><a href="/pages/index.html">home</a></li>
+          <li><a href="/pages/portfolio.html">portfolio</a></li>
+          <li><a href="/pages/about.html">about</a></li>
+        </ul>
+      </nav>
+  </header>
+  <main>
+    <section>
+      <h2>jae clayton: about me</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Id volutpat lacus laoreet non. Magna etiam tempor orci eu lobortis. Morbi tristique senectus et netus et malesuada. Quis blandit turpis cursus in hac habitasse platea dictumst. Donec enim diam vulputate ut. Quis varius quam quisque id diam. Ultricies tristique nulla aliquet enim tortor at auctor urna. Commodo viverra maecenas accumsan lacus vel facilisis volutpat est velit. Dolor sit amet consectetur adipiscing elit ut. Ut pharetra sit amet aliquam id diam maecenas ultricies mi. Amet porttitor eget dolor morbi non. Consequat mauris nunc congue nisi vitae suscipit tellus mauris. Ullamcorper dignissim cras tincidunt lobortis feugiat. Duis ut diam quam nulla porttitor massa id. Ultrices eros in cursus turpis. Commodo ullamcorper a lacus vestibulum sed. Tincidunt eget nullam non nisi est sit amet. Erat imperdiet sed euismod nisi porta. Elit pellentesque habitant morbi tristique senectus et netus et.</p>
+    </section>
+    <section>
```

Comment:
There was a validation warning here about not having a heading in this section. I would say the section itself isn't that necessary here. Think of section as being something that lets us express the logical structure of the content, such that if we gathered all the section headings, we'd be able to build a table of contents. Since this section doesn't really carry any semantic value, I'd replace it with a div, or see if I could remove it entirely.